README.rst

```diff
@@ -16,6 +51,12 @@ to run Telegram bot (which chats with user)
 
 to launch REST API server, for other application to send message to user
 
+TelegramBot
+===========
+In your telegram , we need create telegrambot to get Token in alarmbot projects.
```

Comment:
Why "telegram" name need to be capitalized but you don't, where as "token" does not and you did?